models/models.js

```diff
@@ -8,7 +8,10 @@ exports.fetchTopics = () => {
 
 exports.fetchArticleId = (id) => {
   return db
-    .query(`SELECT * FROM articles WHERE article_id = $1;`, [id])
+    .query(
+      ` SELECT articles.*, COUNT(comment_id) AS comment_count FROM articles LEFT OUTER JOIN comments ON articles.article_id = comments.article_id WHERE articles.article_id = $1 GROUP BY articles.article_id`,
```

Comment:
Have a look at how we can cast a data type into another data type in an SQL Query